cli/cmd/encore/secrets/list.go

```diff
@@ -0,0 +1,164 @@
+package secrets
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+
+	"encr.dev/cli/cmd/encore/cmdutil"
+	"encr.dev/cli/internal/platform"
+	"encr.dev/cli/internal/platform/gql"
+)
+
+var listSecretCmd = &cobra.Command{
+	Use:                   "list [keys...]",
+	Short:                 "Lists secrets, optionally for a specific key",
+	DisableFlagsInUseLine: true,
+	Run: func(cmd *cobra.Command, args []string) {
+		appSlug := cmdutil.AppSlug()
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		defer cancel()
+
+		var keys []string
+		if len(args) > 0 {
+			keys = args
+		}
+		secrets, err := platform.ListSecretGroups(ctx, appSlug, keys)
+		if err != nil {
+			cmdutil.Fatal(err)
+		}
+
+		if keys == nil {
+			// Print secrets overview
+			var buf bytes.Buffer
+			w := tabwriter.NewWriter(&buf, 0, 0, 3, ' ', tabwriter.StripEscape)
+
+			fmt.Fprint(w, "Secret Key\tProduction\tDevelopment\tLocal\tPreview\tSpecific Envs\t\n")
+			const (
+				checkYes = "\u2713"
+				checkNo  = "\u2717"
```

Comment:
Might we have issues with some terminals with these unicode's?

The `errslist` has two different formatters (unicode and ascii) to try and combat this issue - but I guess we can leave it until we get confirmed errors from people